hopsworks-api/src/main/java/io/hops/hopsworks/api/jobs/JobsResource.java

```diff
@@ -1,7 +1,4 @@
 /*
- * Changes to this file committed after and not including commit-id: ccc0d2c5f9a5ac661e60e6eaf138de7889928b8b
```

Comment:
@robzor92 why the license modification? 
@SirOibaf if a file is renamed, we keep the old license?

Reply:
Had to modify the license otherwise the tests failed. https://bbc1.sics.se/jenkins/job/HopsWorks-testing/901/console

Reply:
@tkakantousis If you run **git log --follow hopsworks-api/src/main/java/io/hops/hopsworks/api/jobs/JobsResource.java**

You will see the file originates from commit 9a74acdf0e4785d1d11456fb4e73554940e41026, so the license checker which uses git log to determine if when the file was included thinks only the new license applies to it. Although it have been renamed and had some content modified.

Reply:
You renamed the file, but you also rewrote most (if not all) of the content. Git detects it as new file and I'd argue that you are not using anymore substantial parts of the MIT codebase as both the api spec and actual code has been rewritten.
In this case we can remove the double header.

Next time, please squash your commits before run the tests.

Reply:
@SirOibaf Do you know if there a threshold after which git detects it as a new file? Seems kind of arbitrary.
The fact I didn't squash before, made the check_license not complain when HOPSWORKS-510 was merged?